cfgov/v1/jinja2/v1/home_page.html

```diff
@@ -7,39 +7,54 @@
     {{ _('Consumer Financial Protection Bureau') }}
 {%- endblock %}
 
-{% block content_main %}
-
-<h1 class="u-visually-hidden">
-	{{ _('Consumer Financial Protection Bureau') }} {{ _('homepage') }}
-</h1>
-
-{% for block in page.content %}
-    {% if block.block_type == 'features' %}
-        <section class="wrapper
-                        wrapper__match-content
-                        block
-                        block__bg
-                        block__border
-                        block__flush-top
-                        u-mb0">
+{% block css %}
+    {{ super() }}
+    <style>
+        .m-hero_wrapper {
+            /* #b4b5b6 is @gray-40 */
+            border-right: 1px solid #b4b5b6;
+            border-bottom: 1px solid #b4b5b6;
+            border-left: 1px solid #b4b5b6;
+        }
+    </style>
+{% endblock css %}
+
+{% block hero %}
+    {% for block in page.content %}
+        {% if block.block_type == 'features' %}
+            <section class="wrapper
+                            wrapper__match-content
+                            block
+                            block__bg
+                            block__border
+                            block__flush-top
+                            u-mb0">
+                {% include_block block %}
+            </section>
+        {% else %}
             {% include_block block %}
+        {% endif %}
+    {% endfor %}
+{% endblock hero %}
+
+{% block content_main %}
+    {% if carousel_items %}
+        <h1 class="u-visually-hidden">
+            {{ _('Consumer Financial Protection Bureau') }} {{ _('homepage') }}
+        </h1>
+        {{ carousel.render( carousel_items ) }}
+        <section class="block">
+            {% with value = {'info_units': info_units, 'format': '33-33-33'} %}
+                {% include 'organisms/info-unit-group-2.html' %}
+            {% endwith %}
         </section>
     {% else %}
-        {% include_block block %}
+        {% with value = {'info_units': info_units, 'format': '33-33-33'} %}
+            {% include 'organisms/info-unit-group-2.html' %}
+        {% endwith %}
     {% endif %}
```

Comment:
@Scotchester @anselmbradford Instead of adding a feature flag I'm using the `{% if carousel_items %}` conditional to control the layout.